announcements/triage-wg-welcome.md

```diff
@@ -0,0 +1,22 @@
+The Release team is pleased to announce the **triage working group**, which
+will assume responsibility for triaging issues and pull requests in the
```

Comment:
I would note here that this is taking responsibility from the release team just to show who is currently in charge of this